lib/doorkeeper/openid_connect/helpers/controller.rb

```diff
@@ -97,16 +96,21 @@ def handle_oidc_max_age_param!(owner)
           end
         end
 
-        def reauthenticate_oidc_resource_owner(owner)
+        def return_without_prompt_param(prompt_value)
```

Comment:
Worried about name clashes again:

```suggestion
        def return_without_oidc_prompt_param(prompt_value)
```

(maybe at some point we can move all these helper methods into a module that doesn't get included, so this stops being a concern :wink:)